content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -490,17 +490,17 @@ Validation Error Count: 1
 
 ### <inlinecode>db pull</inlinecode>
 
-<Admonition type="info">
+The `db pull` command connects to your database and adds Prisma models to your Prisma schema that reflect the current database schema.
 
-Introspection with the `db pull` command on the [MongoDB connector](/concepts/database-connectors/mongodb) samples the data instead of reading a schema.
+<Admonition type="warning">
 
-</Admonition>
+**Warning**: The command will overwrite the current `schema.prisma` file with the new schema. Some manual changes or customization can be lost. Be sure to back up your current `schema.prisma` file (or commit your current state to e.g. Git to be able to revert any changes) before running `db pull` if it contains important modifications.
```

Comment:
Makes the wording of this more correct (seems the description was from before Re-Introspection was a thing) and also add a mention of Git instead of backup.